towerlib/entities/job.py

```diff
@@ -1570,8 +1581,7 @@ def project(self):
             Project: The project of the update.
 
         """
-        url = self._data.get('related', {}).get('project')
-        return self._tower._get_object_by_url('Project', url)  # pylint: disable=protected-access
+        return self._tower.get_project_by_id(self._data.get('project'))  # pylint: disable=protected-access
```

Comment:
Are we sure that this actually returns what we expect? The project is mentioned both on the related attributes and as a root attribute?

Reply:
I have tested implemented as the way you suggested. Looks to be working.